src/main/scala/io/iohk/ethereum/domain/Account.scala

```diff
@@ -0,0 +1,16 @@
+package io.iohk.ethereum.domain
+
+import akka.util.ByteString
+import org.spongycastle.util.encoders.Hex
+
+case class Account(nonce: BigInt, balance: BigInt, storageRoot: ByteString, codeHash: ByteString) {
+  override def toString: String = {
+    s"""Account{
+         |nonce: $nonce
+         |balance: $balance wei
+         |storageRoot: ${Hex.toHexString(storageRoot.toArray[Byte])}
+         |codeHash: ${Hex.toHexString(codeHash.toArray[Byte])}
+         |}
+       """.stripMargin
+  }
```

Comment:
This syntax is used everywhere else and it's just a comment. Let's keep it as it is. Summoning the author - @adamsmo - learn about Radek's objections.